scripts/monitornodes.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Monitor load average on nodes in a Selenium grid.
+# Runs forever, Control-C to quit.
+
+set -e
+
+if [ $# -gt 1 ]; then
+    HUB_URL=$1
+else
+    HUB_URL="http://localhost:4444/wd/hub"
+fi
+
+HUB_CONSOLE_URL="$(echo "$HUB_URL" | sed 's,/wd/hub,/grid/console,')"
+
+if [ -z "$JITSI_SSH_CONFIG" ]
```

Comment:
The default path here is what I use in my dev env and I don't know if people are okay with such a default. We could remove it if we wanted to be less opinionated.

Reply:
Any default, or no default, would be fine for me.